website/content/docs/ecs/architecture.mdx

```diff
@@ -9,37 +9,51 @@ description: >-
 
 ![Consul on ECS Architecture](/img/consul-ecs-arch.png)
 
```

Comment:
It might align better with reader expectations to move the description of the diagram above it, e.g.:
"The following diagram shows the main components of the Consul architecture when deployed to an ECS cluster:"

I would keep the bulleted list describing the components below the graphic. 

Just from a reader psychology perspective, this would set up a "here's what to expect", "here's the thing", "here's more info about the thing" format.